### PR TITLE
Move the map down to avoid "hitting" it with the scroll wheel

### PR DESCRIPTION
**Pull request by tjguk**
As an idea to avoid the problem in https://github.com/PyconUK/pyconuk.github.io/issues/7 this PR moves the map to the bottom. Feel free to take or leave!

### CODE REVIEW

File: venue/index.html [hunk 1]
```diff
@@ -89,6 +88,7 @@ <h1 class="huge">Venue</h1>
             </div>
         </div>
     </article>
+    <iframe width="100%" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="http://www.openstreetmap.org/export/embed.html?bbox=-1.5059745311737058%2C52.399833211725685%2C-1.5017688274383545%2C52.40604509180987&amp;layer=mapnik&amp;marker=52.40293598823974%2C-1.5038716793060303" style="border: none"></iframe>
```
Comment (ntoll) [inline venue/index.html]: Shouldn't the iframe still be within the enclosing div (so it remains part of the article)..? For example, could it / should it be on line 89..?

**Comment by hawkz** [comment]
Would it be better to switch to leafletjs instead then do something like:
map.once('focus', function() { map.scrollWheelZoom.enable(); });

**Comment by ntoll** [comment]
@hawkz if that fixes the problem with the path of least resistance then "yes"... :-)

**Comment by inglesp** [comment]
Closing, as this is covered by https://github.com/PyconUK/pyconuk.org/issues/1.

**Pull request closed without merging**